connectors/cli.py

```diff
@@ -45,9 +45,10 @@ def _parser():
     )
 
     parser.add_argument(
-        "--debug",
-        action="store_true",
-        default=False,
+        "--log-level",
+        type=str,
+        choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
+        default=None,
         help="Run the event loop in debug mode.",
```

Comment:
```suggestion
        help="The log level of the application.",
```

Reply:
Yup, noticed it as well :) Thanks for attentive review!